### PR TITLE
Tartare log - Adding a file log rotate

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

File: docker-compose.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ services:
       - /tmp/tartare/input:/var/tartare/input:rw
       - /tmp/tartare/output:/var/tartare/output:rw
       - /tmp/tartare/current:/var/tartare/current:rw
+      - /tmp/tartare/log:/var/log/tartare:rw
```
Comment (antoine-de) [inline docker-compose.yml]: why do you want to share this ?

Comment (prhod) [inline docker-compose.yml]: I thought it was consistant to export logs. You think it would be best to keep the log in the docker?

**Comment by antoine-de** [comment]
I close this I think this has to be handled either by docker or by syslog

**Comment by prhod** [comment]
ok :)

**Pull request closed without merging**